Products/CMFPlone/Portal.py

```diff
@@ -221,9 +221,4 @@ def isEffective(self, date):
         # Override DefaultDublinCoreImpl's test, since we are always viewable.
         return 1
 
-
-# Remove the IContentish interface so we don't listen to events that won't
-# apply to the site root, ie handleUidAnnotationEvent
-classImplementsOnly(PloneSite, implementedBy(PloneSite) - IContentish)
```

Comment:
Please remove imports of `classImplementsOnly`, `implementedBy` and `IContentish`